src/Stripe.Portable/Stripe.Portable.csproj

```diff
@@ -97,6 +97,9 @@
     <Compile Include="..\Stripe\Entities\StripeCharge.cs">
       <Link>Entities\StripeCharge.cs</Link>
     </Compile>
+    <Compile Include="..\Stripe\Entities\StripeCountrySpec.cs">
```

Comment:
This should be renamed "CountrySpec" - see the top of my readme for the direction I am taking all new services (7.0 will have everything changed)